src/containers/projects/filters.svelte

```diff
@@ -0,0 +1,56 @@
+<script>
+  import TextField from 'ui/text-field.svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import Dot from 'ui/dot.svelte';
+  import Button from 'ui/button.svelte';
+  import RadioGroup from 'ui/radio-group.svelte';
+
+  import Spots from '@/components/projects/spots.svelte';
+  import Competences from '@/components/projects/competences.svelte';
+  import DateRange from '@/components/projects/date.svelte';
+
+  import orders from '@/constants/projects/order';
+  import {selectedOrder, selectOrder, filters, changeSearch} from '@/store/projects';
+
+  let open = null;
+  const changeOpen = (name) => {
+    if (open !== name || !open) open = name;
```

Comment:
if `open` was null or undefined, it will not be equal to `name`. Therefore, the `|| !open` is not needed imho.